docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -35,78 +36,72 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
+  * `Microsoft Visual Studio 2019 with MSBuild <http://visualstudio.microsoft.com/downloads/>`_
+  * `CMake 3.14 or higher, 64-bit <https://cmake.org/download/>`_ (optional, only required for building sample applications)
+  * `Python 3.6 - 3.10, 64-bit <https://www.python.org/downloads/windows/>`_
+     * Note that OpenVINO is gradually phasing out support for Python 3.6. Python 3.7 - 3.10 are recommended.
 
   .. note::
-    You can choose to download Community version. Use `Microsoft Visual Studio installation guide <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`_ to walk you through the installation. During installation in the **Workloads** tab, choose **Desktop development with C++**.
+     To install Microsoft Visual Studio 2019, follow the `Microsoft Visual Studio installation guide <https://docs.microsoft.com/en-us/visualstudio/install/install-visual-studio?view=vs-2019>`_. You can choose to download the Community version. During installation in the **Workloads** tab, choose **Desktop development with C++**.
 
   .. note::
     You can either use `cmake<version>.msi` which is the installation wizard or `cmake<version>.zip` where you have to go into the `bin` folder and then manually add the path to environmental variables.
   
   .. important::
-    As part of this installation, make sure you click the option **Add Python 3.x to PATH** to `add Python <https://docs.python.org/3/using/windows.html#installation-steps>`_ to your `PATH` environment variable.
+    When installing Python, make sure you click the option **Add Python 3.x to PATH** to `add Python <https://docs.python.org/3/using/windows.html#installation-steps>`_ to your `PATH` environment variable.
 
 @endsphinxdirective
 
 ## Installing OpenVINO Runtime
 
 ### <a name="install-openvino"></a>Step 1: Download and Install OpenVINO Core Components
 
-1. Select and download the OpenVINO™ archive files from [Intel® Distribution of OpenVINO™ toolkit download page](https://software.intel.com/en-us/openvino-toolkit/choose-download).
-   There are typically two files for you to download: 
+1. Open a command terminal as administrator by searching for "Command Terminal" in the Start menu, right clicking on it, and selecting "Run as administrator". Create a folder for OpenVINO and move into it by issuing the following commands.
    ```sh
-   w_openvino_toolkit_<operating system>_<release version>_<package ID>_x86_64.zip
-   w_openvino_toolkit_<operating system>_<release version>_<package ID>_x86_64.zip.sha256
-   ``` 
-   where the `.sha256` file is used to verify the success of the download process.
-   
-2. Locate the downloaded files in your system. This document assumes the files are in your `Downloads` directory. 
-   
-3. Open a command prompt terminal window, and run the following command with the `.zip` file:
-   ```sh
-   CertUtil -hashfile <archive name>.zip SHA256
```

Comment:
do you think we can keep this section under a spoiler?

Reply:
Just want to check, what is a spoiler?

Reply:
Sorry, I meant collapsible section like here https://github.com/openvinotoolkit/openvino/blob/master/docs/OV_Runtime_UG/deployment/deployment-manager-tool.md#running-deployment-manager-in-interactive-mode

Reply:
@ilya-lavrenov Thanks for the suggestion. I didn't manage to create the collapsible block, so to save some time investigating, I included this info in a note. Hope that's good with you. If you don't mind, can I merge this PR?